DistributionPackages/Neos.NeosIo/Resources/Private/Scss/Utilities/_Neos.scss

```diff
@@ -4,12 +4,18 @@
 }
 .neos-alignment-left {
 	float: left;
+	margin-right: .9rem;
 }
 .neos-alignment-center {
 	text-align: center;
 }
 .neos-alignment-right {
 	float: right;
+	margin-left: .9rem;
+
+	figcaption {
+		margin-left: .9rem;
+	}
```

Comment:
Have you checked how it will display now on mobile devices? It might be better to introduce the margins only on larger viewports.

Reply:
Yes. Should be looking fine now !